### ISSUE TITLE
Placing image above or below the content

### ISSUE THREAD

**Issue by robertgarrigos** [backdrop-contrib/iivblock#3]
It would be nice to be able to put the image above or below the content.

**Comment by ininfoveritas** [comment]
Hello,
Thank you for this idea.
I will take a look at this in the next days.

(I don't know how pull request works but internet is my friend and i will find :-) )

**Comment by ininfoveritas** [comment]
I made some small changes to div, row and col. This seems to work better for me?
Does this ok for you?

**Comment by robertgarrigos** [comment]
The only problem I have is that without the classes `d-flex justify-content-center` the image doesn't get centered.

**Comment by ininfoveritas** [comment]
Hello,
What theme do you work on ?
Your version is not working for me with IIV Dark and Basis (default theme).
The modification i made are working on this two themes.

**Comment by robertgarrigos** [comment]
Yes, yes, my fault. I didn't realize that I was on a bootstrap 5 theme...
